lib/sass/script/functions.rb

```diff
@@ -1670,6 +1687,42 @@ def feature_exists(feature)
     end
     declare :feature_exists, [:feature]
 
+    # Returns a reference to a function for later invocation with the `call` function.
+    #
+    # The function reference created may refer to a function defined in
+    # your stylesheet, built-in to the host environment, or a CSS native
+    # function. As such, this function never returns an error, but the
+    # you can use `function-exists()` to check if the function reference
+    # created points at a Sass function.
+    #
+    # @example
+    #   function-reference(rgb)
+    #
+    #   @function myfunc { @return "something"; }
+    #   function-reference(myfunc)
+    #
+    # @overload function_reference($name)
+    #   @param name [Sass::Script::Value::String] The name of the function being referenced.
```

Comment:
Document that a function reference can also be passed and will be returned as-is.